README.md

```diff
@@ -60,14 +52,101 @@ Spinal cord MRI data:
 - T2star tra
 Brain MRI data:
 - DWI (A-P and P-A phase encoding)
-- MPM (multi-parameter mapping)
+- T1w
 
 ## naming convention
 
 sub-<site><pathology>XXX
 
 example:
-sub-zurichDCM001
+sub-montrealDCM001
+
+<ADDITIONAL NOTES IF AVAILABLE>Note: the label `bp-cspine` is used to differentiate spine images from brain.
+
+## derivatives
+
+The derivatives will be organized according to the following:
+
+https://github.com/ivadomed/ivadomed/wiki/repositories#derivatives
+
+Convention for derivatives JSON metadata:
+
+{
+  "Author": "Firstname Lastname",
+  "Date": "YEAR-MM-DD HH:MM:SS"
+}
+
+## <IF DATA ARE MISSING FOR SOME SUBJECT(S)>missing data
+
+```
+
+</details>
+
+## `dataset_description.json`
+
+The `dataset_description.json` is a JSON file describing the dataset.
+
+<details><summary>dataset_description.json template:</summary>
+
+```json
+{
+    "BIDSVersion": "BIDS 1.8.0",
+    "Name": "<dataset_name>"
+}
+```
+
+</details>
+
+## `participants.tsv`
+
+The `participants.tsv` is a TSV file and should include the following columns:
+
+| participant_id | pathology | data_id | institution_id | institution |
+| ----------- | ----------- | ----------- | ----------- | ----------- |
+| sub-001 | HC | 001 | 01 | montreal |
+
+- `participant_id` - unique participant ID
+- `pathology` - pathology name; can take values listed in the [pathology column](https://docs.google.com/spreadsheets/d/1yjcA8Z0COn4OZxusIDHjStH2DpeXvscsj-aWE2X-_sg/edit?usp=sharing)
+- `data_id` -  subject ID used to locate the unprocessed data as under `duke/mri/`
+- `institution_id` - institution ID used to locate the unprocessed data as under `duke/mri/`
```

Comment:
We should not refer to `duke` anywhere here. `duke` will eventually go away. These datasets should be self-contained.

`smb://duke.neuro.polymtl.ca/`, even the more restricted `smb://duke.neuro.polymtl.ca/mri/` subset, is not organized well enough for this ID scheme to work in my opinion. Here's a random sampling of some of the subject IDs from it:

```
p115628@joplin:~$ find /mnt/duke/mri/ -mindepth 2 -maxdepth 2 -type d | grep -v ukbiobank | sort -R | head
/mnt/duke/mri/hcp/857263
/mnt/duke/mri/hcp/597869
/mnt/duke/mri/ucsf/Myelopathy_Data
/mnt/duke/mri/hcp/679770
/mnt/duke/mri/sct-users/20150723_SpinalCord_1
/mnt/duke/mri/hcp/107018
/mnt/duke/mri/hcp/114621
/mnt/duke/mri/hcp/163331
/mnt/duke/mri/ucl/spine-generic
/mnt/duke/mri/biospective/SpinalCord 3 2015-07-23
```

So would that be `hcp-107018` might be a sensible enough ID, but `ucsf-Myelopathy_Data` would be silly.

Why do we need to keep this mapping at all? Isn't that kind of sketchy? In an anonymized dataset, doesn't keeping that risk breaking anonymization?

Where we must keep the mapping, the [YODA](https://handbook.datalad.org/en/latest/basics/101-127-yoda.html) plan would say to upload a separate `git-annex` repo to the same server and then use e.g. `git submodule add rawdata ssh://data.neuro.polymtl.ca/raw/my-raw-dataset`. Or, if not `git submodule`, then [at least add `git clone -b v1.0.whatever ssh://data.neuro.polymtl.ca/raw/my-raw-dataset rawdata` [to the curation script](https://github.com/neuropoly/data-management/issues/136#issuecomment-941575717) as kept under `code/`.

Reply:
> We should not refer to duke anywhere here. duke will eventually go away. These datasets should be self-contained.

I think the idea here is to keep provenance. For example, if researchers share non-curated data in DICOM format, they would go under `duke/mri`. Then, we would BIDSify it, put it under data.neuro.polymtl.ca and in order not to miss the provenance of, let's say, `sub-001` (which corresponds to e.g: `MS032_FLASH_T2w-bottom-sag_FA23deg.dcm`).